### DISCUSSION TITLE
Modal image sizing Portrait and Landscape with zoom

### DISCUSSION THREAD

**Discussion by vapordev** [tailwindlabs/tailwindcss#8328] [Help]
We are wanting to display an image in a modal that either shows Portrait Full Height, or Landscape Full Width. Then if you click the image, it zooms Full Screen.

**Comment by DanielOberlechner** [comment]
Thats the job for a JS Library ... or the furthest I can think is the aspect ratio plugin from tailwindcss:
https://github.com/tailwindlabs/tailwindcss-aspect-ratio

But what is the problem why did you create this discussion?!
